backend/config/corpora-api.yml

```diff
@@ -151,12 +151,15 @@ paths:
       summary: Delete a collection
       security:
         - cxguserCookie: [ ]
+        - dummyAuth: [ ]
       description: >-
         Deletes an entire private collection from cellxgene data-portal, including any generated artifacts/assets and
         deployments.
-      operationId: corpora.lambdas.api.v1.collection_uuid.delete
+      operationId: corpora.lambdas.api.v1.collection.delete_collection
       parameters:
         - $ref: "#/components/parameters/path_collection_uuid"
+        - $ref: "#/components/parameters/query_visibility"
```

Comment:
this endpoint should only work for private collections. Why do we need visibility?

Reply:
To allow for a clear error message/that the user was specific about what they were trying to do and to avoid an unnecessary db call. But I can remove it